src/Core/Common/UseParallelizeImageRegion/Documentation.rst

```diff
@@ -0,0 +1,32 @@
+Duplicate An Image
+==================
+
+.. index::
+   single: MultiThreaderBase
+
+Synopsis
+--------
+
+
+This example demonstrates how to take advantage of MultiThreaderBase::ParallelizeImageRegion.
+A comparison is made with LogImageFilter for purpose of computing log(1+x), where x is pixel value.
+
+
+Results
+-------
+
```

Comment:
We could count the number of pixel that are different, then print to `stdout` *LogImageFilter and ParallelizeImageRegion generate the same result.*

Reply:
Done